src/azure-cli/azure/cli/command_modules/identity/_params.py

```diff
@@ -22,12 +22,14 @@ def load_arguments(self, _):
         c.argument('location', get_location_type(self.cli_ctx), required=False)
         c.argument('tags', tags_type)
 
-    with self.argument_context('identity federated-credential', min_api='2022-01-31-preview') as c:
-        c.argument('federated_credential_name', options_list=('--name', '-n'), help='The name of the federated identity credential resource.')
+    with self.argument_context('identity federated-credential', min_api='2025-01-31-preview') as c:
+        c.argument('federated_credential_name', options_list=['--fc-name', '-f'], help='The name of the federated identity credential resource.')
```

Comment:
Renaming `federated_credential_name` from `--name`/`-n` to `--fc-name`/`-f` is a breaking change and not allowed.